thrust/cmake/README_CMake.md

```diff
@@ -0,0 +1,211 @@
+# Using Thrust with CMake
+
+Thrust provides configuration files that simplify the process of using Thrust
+from other CMake projects. Requirements:
+
+- Thrust >= 1.9.10
+- CMake >= 3.10
+
+## Simple Usage
+
+For most projects, the following should be sufficient:
+
+```$cmake
+find_package(Thrust REQUIRED CONFIG)
+
+add_executable(MyProgram ${MySources})
+target_link_libraries(MyProgram Thrust::Thrust)
+```
+
+*Note:* If CMake is unable to locate Thrust, specify the path to Thrust's CMake
+configuration directory as `Thrust_DIR`, e.g.:
+
+```
+$ cmake . -DThrust_DIR=/usr/local/cuda/include/thrust/cmake/
+```
+
+## Configuration Options
+
+The Thrust targets can be configured using the following CMake options:
+
+#### `THRUST_HOST_SYSTEM`
+
+- Configures the host system for the `Thrust::Thrust` CMake target.
+- Possible values are stored in `THRUST_HOST_SYSTEM_OPTIONS`:
+  - `CPP`: The Thrust-provided serial backend (default)
+  - `OMP`: The OpenMP backend
+  - `TBB`: The TBB backend
+  - `NONE`: Does not configure any host system for `Thrust::Thrust`
```

Comment:
`NONE` is slightly misleading, since the code already has some defaults. If this is meant to just configure the linking part and not specify options explicitly (which _still_ indicates some defaults), `MANUAL` may be better.

Reply:
This option is no longer needed with the new function based API, another perk.